### PR TITLE
Add a "Stop" task

### PR DESCRIPTION
**Pull request by sschuberth**
Note that I haven't been able to actually test this code yet, but still I think it makes sense to start reviewing it.

### CODE REVIEW

File: gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/Start.groovy [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package com.novoda.gradle.command
+
+class Start extends Run {
```
Comment (stefanhoth) [inline gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/Start.groovy]: This looks like it's accidentally included?

**Comment by stefanhoth** [comment]
@sschuberth Thank you for your contribution. If you could remove that empty class I'm happy to merge your PR.

**Comment by devisnik** [comment]
Looks good, thanks!
Would be great if you could wire the task in the plugin class:
https://github.com/novoda/gradle-android-command-plugin/blob/develop/gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPlugin.groovy#L19

File: gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/Start.groovy (hunk 1 above)
Comment (devisnik) [inline gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/Start.groovy]: Might make sense to have a `start` alias for run. Should also be wire by default, if we want to go with it.

Comment (friedger) [inline gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/Start.groovy]: We could wire a stop task alias without the extra class
Am 25.06.2015 18:20 schrieb "Volker Leck" <notifications@github.com>:

**Comment by sschuberth** [comment]
Thanks for the suggestions. I've addressed the issues and force-pushed a new iteration of the PR.

File: gradle-android-command-plugin/src/test/groovy/com/novoda/gradle/command/AndroidCommandPluginExtensionTest.groovy [hunk 2]
```diff
@@ -26,7 +26,7 @@ class AndroidCommandPluginExtensionTest extends GroovyTestCase {
     }
 
     private static AndroidCommandPluginExtension createExtension() {
-	def projectDir = new File('..')
+        def projectDir = new File('..')
```
Comment (xrigau) [inline gradle-android-command-plugin/src/test/groovy/com/novoda/gradle/command/AndroidCommandPluginExtensionTest.groovy]: :+1:

**Comment by xrigau** [comment]
@stefanhoth :shipit: ?

**Comment by stefanhoth** [comment]
Did someone try if it works?

**Comment by sschuberth** [comment]
I did try the `stop` task and it worked. I did not try the `start` task alias yet because neither `run` nor `start` work for our app which uses an `activity-alias` as the launcher activity. I'm currently also working on a fix for this.

**Comment by devisnik** [comment]
:+1: 
@sschuberth We have a sample project under `sample/`. That's wired to the dev version of the plugin automagically. So you can just test with that.

**Comment by sschuberth** [comment]
Ah, thanks. I tried with `./gradlew sample:startFreeBetaDebug` and `./gradlew sample:stopFreeBetaDebug`, both working as expected.

PS: Maybe you should wire up some Travis CI job to automatically test these?

**Comment by friedger** [comment]
We have https://ci.novoda.com/job/gradle-android-command-plugin/

**Comment by sschuberth** [comment]
@friedger But that does not seem to be hooked up with GitHub PRs, or? It would be nice to get some feedback from the CI in the PR's web page.

**Comment by stefanhoth** [comment]
@sschuberth I just set up the PR builder on our CI
https://ci.novoda.com/job/gradle-android-command-plugin-prb/

Waiting for your next contribution to test it. :wink:

**Comment by devisnik** [comment]
:+1: Keep 'em coming. ;)

**Comment by sschuberth** [comment]
Here you go: https://github.com/novoda/gradle-android-command-plugin/pull/79 ;-)

**Pull request merged**